screens/FollowerList.tsx

```diff
@@ -0,0 +1,179 @@
+import { useState } from "react";
+import {
+  View,
+  Text,
+  Image,
+  FlatList,
+  TouchableOpacity,
+  StyleSheet,
+  ScrollView,
+} from "react-native";
+import { SHADOWS, SIZES, COLORS } from "../constants/theme";
+import { useRoute } from "@react-navigation/native";
+import { Ionicons } from "@expo/vector-icons";
+import { StackNavigationProp } from "@react-navigation/stack";
+
+type User = {
+  id: string;
+  username: string;
+  nickname: string;
+  avatar: string;
+  posts: string[];
```

Comment:
sollte es nicht eine Post Componente geben, von welcher wir das Array haben wollen? Anstatt eines String arrays?

Reply:
ja, haste recht. Aber die Posts haben wir aktuell noch nicht. Wir haben generell kein Attribut, bei der UserInfo mit seinen Posts, weil wir uns ja auch drauf geeinigt haben, in dieser Iteration keine Bilder zu machen. Daher jetzt erstmal diese Version mit dem Dummy string array